.github/pull_request_assignment.yml

```diff
@@ -65,6 +65,26 @@
       - jhendrixMSFT
       - tjprescott
 
+- rule:
+    paths:
+        - "specification/**/*Management/**/*.cadl"
+        - "specification/**/*Shared/**/*.cadl"
```

Comment:
Shared would include data plane shared library. It should be `*.Management.Shared`. Can you also add another set of path for tsp extension?

This would include the services onboarded to shift left. Let's check in scrum if reviewers are ok to review them.

Reply:
Path should add another one for resource-manager/cadl/* which is existed cadl mgmt. services.

Reply:
Added tsp extension and resource-manager/cadl